### PR TITLE
TMEDIA-696 - Add Image component

### PR DESCRIPTION
**Pull request by matthewroach**
Added Image component, tests, scss hooks, and documentation

**Note**: This is just a basic component for now to unblock upcoming work - resizer implementation for images will be done at a later date

### Ticket 
[TMEDIA-696](https://arcpublishing.atlassian.net/browse/TMEDIA-696)

### Testing
* Checkout branch - `git checkout TMEDIA-696-image-component`
* Run storybook - `npm run storybook`
* Check storybook output examples and documentation
* Check chromatic build

### CODE REVIEW

File: src/components/image/index.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,54 @@
+import { Meta, Story, Preview, Props } from "@storybook/addon-docs";
+
+import Image from ".";
+
+<Meta title="Components/Image" component={Image} />
+
+# Image
+
+Image provide a vital part of story telling for websites. Images are used to complement text, or provide supporting information.
+Images can be used as informationaly or for decoration. This component will provide the means to display images in a variety of different ways.
```
Comment (brentmiller1973) [inline src/components/image/index.stories.mdx]: informational instead of informationaly

File: src/components/image/index.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+	<Story name="Default">
+		<Image src="/computer-user.jpeg" alt="A person sat down using a laptop computer" />
+	</Story>
+</Preview>
+
+** Image not found **
+
+<Preview>
+	<Story name="Image not found">
+		<Image src="cat.jpg" alt="A cat jumping up to a ball" />
```
Comment (brentmiller1973) [inline src/components/image/index.stories.mdx:39]: More of a general comment, but eventually, once defined, we may want to include which style tokens are applied to these components in the storybook documentation.  Thinking from a client's perspective, it's not clear how you may want an image's width to be 100% and the height auto, for example.  They may only think that you can control the width and height by pixel values.

File: src/components/image/index.stories.mdx (hunk 1 above)
Comment (matthewroach) [reply src/components/image/index.stories.mdx]: Updated

**Review by brentmiller1973 — APPROVED** [review]

File: src/components/image/index.stories.mdx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+<Canvas>
+	<Story name="Width and Height">
+		<Image
+			src="/computer-user.jpeg"
+			alt="A person sat down using a laptop computer"
+			width="200"
+			height="147"
+		/>
+	</Story>
+</Canvas>
```
Comment (nschubach) [inline src/components/image/index.stories.mdx:54]: Do we want some iterations like width/height only?

```
** Width only **

<Canvas>
	<Story name="Width only">
		<Image
			src="/computer-user.jpeg"
			alt="A person sat down using a laptop computer"
			width="250"
		/>
	</Story>
</Canvas>

** Height only **

<Canvas>
	<Story name="Height only">
		<Image
			src="/computer-user.jpeg"
			alt="A person sat down using a laptop computer"
			height="250"
		/>
	</Story>
</Canvas>
```

**Review by nschubach — APPROVED** [review]
Only one suggestion, but not required

File: src/components/image/index.stories.mdx (hunk 3 above)
Comment (matthewroach) [reply src/components/image/index.stories.mdx:54]: Added @nschubach

**Pull request merged**